docs/architecture/decisions/0006-use-tailwind-css-instead-of-emotion.md

```diff
@@ -0,0 +1,28 @@
+# 6. Use tailwind css instead of emotion
+
+Date: 2021-05-31
+
+## Status
+
+Accepted
+
+Supercedes
+[3. Use nextjs, express, emotion](0003-use-nextjs-express-stylecomponents.md)
+
+## Context
+
+[Tailwindcss](https://tailwindcss.com/) is:
+
+> A utility-first CSS framework packed with classes like flex, pt-4, text-center
+> and rotate-90 that can be composed to build any design, directly in your
+> markup.
+
+## Decision
+
+- We will use `tailwindcss` instead of `emotion` for styling
```

Comment:
So I'm curious -- why not both? From my past experiences with tailwind, it seems like it covers 90% of use-cases, but we need a way to cover that last 10%. It seems like we could accomplish that with Emotion, regular CSS/SCSS, or by extending Tailwind's theme.

Do you agree that we need something in addition to what Tailwind offers out of the box? If so, what would your preference be?

Reply:
What I was thinking is to directly do everything the `tailwaind` way... by modifying the configuration to suit our needs:
https://tailwindcss.com/docs/configuration

My thinking is that it could get messy to combine them both (styled-components/emotion and tailwind). We already have React to help, and I was thinking this would be enough. 

I wanted to avoid any "manual" css outside of the tailwind framework. Do you think this would be unavoidable?

Reply:
I am open to experimenting with `tailwind`. Especially loved the off-the-shelf `dark mode` option that it provides. The only downside I can see right now is initial learning curve to get familiar with class names. 
I agree with Vic..code readability would reduce if we decide to use both `tailwind` and emotion since both of them are within html tags. 
Should we go with just `tailwind` for now and then integrate regular CSS/SCSS only if need arises? that way we would be careful to not unnecessarily use the fallback option.